### PR TITLE
evolution-ews 3.26.2_1

### PR DESCRIPTION
**Pull request by chama-chomo**
EWS module for Evolution

### CODE REVIEW

**Comment by pullmoll** [comment]
Please squash commits and use `git commit --amend` followed by `git push -f` for updates.

**Comment by chama-chomo** [comment]
I'm trying to squash, though not sure I was successful. I apologize, this is my first day with git.

File: srcpkgs/evolution-ews/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'evolution-ews'
+pkgname=evolution-ews
+version=3.26.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DLIBEXEC_INSTALL_DIR=/usr/lib/evolution -DCMAKE_BUILD_TYPE=Release"
```
Comment (pullmoll) [inline srcpkgs/evolution-ews/template]: CMAKE_INSTALL_PREFIX and CMAKE_BUILD_TYPE are already defined by the build style cmake. You can remove them from configure_args.

**Comment by pullmoll** [comment]
You can `git pull --rebase` to avoid the `Merg branch 'master' ...` commits.

**Pull request closed without merging**